base/threadingconstructs.jl

```diff
@@ -103,22 +103,28 @@ Create and run a [`Task`](@ref) on any available thread. To wait for the task to
 finish, call [`wait`](@ref) on the result of this macro, or call [`fetch`](@ref)
 to wait and then obtain its return value.
 
+(Values can be interpolated into `@spawn` via `\$` to evaluate them in the current task.)
```

Comment:
Should be a `!!! compat` note

Reply:
Done, thanks. Is it okay that I added the compat note below like this, or should it go above the section on `$`-interpolation? Should I move that paragraph below the compat notes?